android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -89,6 +95,23 @@ public Snapshot(View view) {
     originY = location[1];
     width = view.getWidth();
     height = view.getHeight();
+
+    View transformedView = findTransformedView(view);
+    if (transformedView != null) {
+      float[] transformMatrixArray = new float[9];
+      transformedView.getMatrix().getValues(transformMatrixArray);
+      transformMatrix = new ArrayList<>();
+      for (int i = 0; i < 9; i++) {
+        transformMatrix.add(transformMatrixArray[i]);
+      }
```

Comment:
You can replace this with
```suggestion
      transformMatrix = new ArrayList<>(Arrays.asList(transformMatrixArray));
```
If you're sure `transformMatrixArray`'s length is 9.
If you don't need `transformMatrix` to be an `ArrayList` (I don't think you do, it just needs to be a `List`), you may even just have
```suggestion
      transformMatrix = Arrays.asList(transformMatrixArray);
```
I'm pretty sure

Reply:
Unfortunately not :/ It would works with `Float[]` but not with `float[]`

Reply:
Right 👍